v1.0.0.yaml

```diff
@@ -224,12 +209,18 @@ paths:
           schema:
             type: string
       responses:
+        "429":
+          $ref: '#/components/responses/RateLimitExceeded'
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+        "403":
+          $ref: '#/components/responses/Forbidden'
         "200":
           description: File details object.
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/fileDetails"
+                $ref: "#/components/responses/FileDetails"
```

Comment:
Note: AFAIK, moving the schema from schemas to responses will lead to Stoplight showing no schema for the response body like this:

<img width="513" alt="image" src="https://github.com/imagekitio/openapi/assets/141143550/d9f03fac-adf2-4d88-8bdd-a7f10e29a284">